todo_app/app.py

```diff
@@ -1,11 +1,32 @@
 from flask import Flask
+from flask import render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+
 
 app = Flask(__name__)
-app.config.from_object(Config())
+app.config.from_object(Config)
+
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    # show all to-do items
+    items = get_items()
+    return render_template("index.html", items=items)
+
+
+@app.route('/New_List', methods=["POST"])
+def New_List():
+    # add new item
+    title = request.form.get("title")
+    #add_item(title)
```

Comment:
Why is this commented out - this is a key line for having this method work!
```suggestion
    add_item(title)
```